driver/src/main/java/oracle/nosql/driver/NoSQLHandle.java

```diff
@@ -230,6 +241,10 @@ public interface NoSQLHandle extends AutoCloseable {
      *
      * @throws NoSQLException if the operation cannot be performed for any other
      * reason
+     *
+     * @since 5.4.16 {@link PutRequest#setReturnRow} will return existing
```

Comment:
Remove the Since here: it's inaccurate. The logic is independent of the SDK version.

Reply:
Removed